### ISSUE TITLE
Wrap PSA

### ISSUE THREAD

**Issue by chrysn** [RIOT-OS/rust-riot-wrappers#69]
Now that we have PSA in RIOT, let's make it available also on the Rust side.

Open issues to solve this:
* [ ] https://github.com/parallaxsecond/rust-psa-crypto/issues/131

Cc'ing @einhornhool (who is the expert on the PSA side) and @geonnave (whose PR that happens to have the same number as the rust-psa-crypto issue, https://github.com/openwsn-berkeley/edhoc-rs/pull/131, will eventually need this) -- I trust you'll unsubscribe if you don't want to track this.

**Comment by geonnave** [comment]
Just to make sure I understand, once solved, we would have the `psa_crypto` crate using RIOT's PSA instead of [another PSA wrapper in C which in turn uses mbedtls] ?

**Comment by chrysn** [comment]
Yes. More precisely, I'd hope that psa_crypto_sys will decide based on the system state (as most sys crates do -- use pkgconfig or explicitly passed paths) to use one or the other.
